pkg/scanners/terraform/parser/parser_test.go

```diff
@@ -38,14 +38,18 @@ moved {
 
 }
 
+import {
+
+}
+
```

Comment:
I see that you've added this but none of the test assertions were modified. How can we test this change?

Reply:
Without the implementation code, the test will fail.
Here, as with the moved block, this means that the import block is not a syntax error.

Like:

```
# Case: Only changed test code
$ git diff origin/master
diff --git a/pkg/scanners/terraform/parser/parser_test.go b/pkg/scanners/terraform/parser/parser_test.go
index 8f693a0d..ebef5458 100644
--- a/pkg/scanners/terraform/parser/parser_test.go
+++ b/pkg/scanners/terraform/parser/parser_test.go
@@ -38,6 +38,10 @@ moved {

 }

+import {
+
+}
+
 resource "cats_cat" "mittens" {
        name = "mittens"
        special = true

$ make test
...
--- FAIL: Test_BasicParsing (0.00s)
    parser_test.go:68: test.tf:19,1-7: Unsupported block type; Blocks of type "import" are not expected here.
```

Reply:
@simar7 Do you mean you want more tests? (For example, for attributes in an import block)

Reply:
> @simar7 Do you mean you want more tests? (For example, for attributes in an import block)

Yes that's right. I meant that we should test this import block functionality by having attributes in the import block.

Reply:
Added there testing from 4ca5e4f